test/test_verify.py

```diff
@@ -17,7 +17,9 @@ def assertVerifyExpectFail(self, *args, **kwargs):
             verify(*args, **kwargs)
         except AssertionError as e:
             if str(e):
-                self.assertExpected(str(e))
+                # substring a small piece of string because the exact message
+                # depends on system's formatting settings
```

Comment:
My guess is it's not system dependent but rather numpy version dependent. We have seen format changed between numpy 1.13.3 and 1.14.0.

Reply:
I think it's best to just trim the message. In this case we just care about general type of the error, not exact message. And reasonable that devs would have different numpy version potentially.

The message is like:
```
ONNX backend returned different results from PyTorch		  ONNX backend returned different results from PyTorch
ERROR: In output 0: Arrays are not almost equal to 3 decimals		
		
(mismatch 100.0%)		
 x: array([-2., -2.], dtype=float32)		
 y: array([4., 6.], dtype=float32)		
		
  * If you are not using trained parameters, a difference in results		
    could mean that your network is numerically unstable.  Otherwise
```